sql/core/src/main/scala/org/apache/spark/sql/execution/basicPhysicalOperators.scala

```diff
@@ -345,6 +345,16 @@ case class RangeExec(range: org.apache.spark.sql.catalyst.plans.logical.Range)
 
   override val output: Seq[Attribute] = range.output
 
+  override def outputOrdering: Seq[SortOrder] = range.outputOrdering
+
+  override def outputPartitioning: Partitioning = {
+    if (numSlices == 1) {
+      SinglePartition
```

Comment:
Which one is better? `SinglePartition` or `RangePartitioning(outputOrdering, 1)`?

Reply:
`SinglePartition` is better